compiler-rt/include/sanitizer/tsan_interface.h

```diff
@@ -126,6 +126,10 @@ void __tsan_mutex_post_signal(void *addr, unsigned flags);
 void __tsan_mutex_pre_divert(void *addr, unsigned flags);
 void __tsan_mutex_post_divert(void *addr, unsigned flags);
 
+// Annotate that no mutexes can be held. If we are holding mutexes, then
```

Comment:
This is effectively not an annotation, but a check. So I would say something along the following lines:

```
// Check that the current thread does not hold any mutexes,
// report a bug report otherwise.
```

Reply:
Will change